### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by jorgecerna**
Just documenting

### CODE REVIEW

**Review by Dvillahermosa310 — APPROVED** [review]

**Review by Dvillahermosa310 — APPROVED** [review]
good

**Review by dromero310 — APPROVED** [review]
Hiyo!

**Comment by jorgecerna** [comment]
Good stuff

**Review by corymaxbernhardt — APPROVED** [review]

**Review by dromero310 — APPROVED** [review]

**Review by corymaxbernhardt — APPROVED** [review]
GREAT!

File: README.md
Comment (dromero310) [inline README.md:1]: FACK!

**Pull request merged**